src/backend/expungeservice/form_filling.py

```diff
@@ -296,7 +306,7 @@ def _set_font_for_field(field, field_value, kid, warnings):
     @staticmethod
     def _build_font_string(field: PdfDict, field_value: str) -> Tuple[str, bool]:
         max_length = FormFilling._compute_field_max_length(field)
-        needs_shrink = len(field_value) > max_length
+        needs_shrink = len(field_value) > max_length if field_value != '/Yes' else False
```

Comment:
What fields have /Yes?

Reply:
* Here are the files generated with John Common:
[OSP_Form_new.pdf](https://github.com/codeforpdx/recordexpungPDX/files/10832498/OSP_Form_new.pdf)
[COMMON A NAME_120000_baker_new.pdf](https://github.com/codeforpdx/recordexpungPDX/files/10832499/COMMON.A.NAME_120000_baker_new.pdf)
[COMMON NAME_110000_baker_new.pdf](https://github.com/codeforpdx/recordexpungPDX/files/10832500/COMMON.NAME_110000_baker_new.pdf)
[COMMON NAME_200000_benton_new.pdf](https://github.com/codeforpdx/recordexpungPDX/files/10832501/COMMON.NAME_200000_benton_new.pdf)

* For the checkbox fields, I had to use `/Yes` for the field values, but they generate the check mark symbol that Adobe programs in. I'm hoping that will fix the related Issue.